SEED-platform/seed
# Post import summary cleanup

#### Any background context you want to provide?
Import results didn't seem to be coming through during the post import summary, especially for matching record counts. Upon more research, I discovered that, while building out the recent matching changes, I didn't fully understand and update the results of the import. 

Also, the file name wasn't showing up when there weren't any matches.

#### What's this PR do?
The results needed to be updated to accommodate for all the possibilities on import. The new possibility was that merges could happen between existing records on import of new records.

Along with that, the variable names were made more descriptive and consistent upon initializing and transferring between BE and FE code.

Lastly, the filename is added to modal reporting matches when there are no matches.

#### How should this be manually tested?
After an import without matches, check that file name appears.

Import records with different combinations of the following:
- with and without matches...
- with and without duplicates..
- within the file being imported
- against existing records
- between existing records (records that were auto merged but were then manually unmerged and unchanged)
Verify the counts during these imports.

Here are the list of possibilities with notes next to some:
- New properties  _(includes records that were in-import-file merges as well)_
- New tax lots  _(includes records that were in-import-file merges as well)_
- Merged properties within import file _(can be included in 'New properties' or even 'Merged properties against existing records')_
- Merged tax lots within import file _(can be included in 'New tax lots' or even 'Merged tax lots against existing records')_
- Merged properties against existing records
- Merged tax lots against existing records
- Merged properties between existing records
- Merged tax lots between existing records
- Ignored property duplicates within the import file
- Ignored tax lot duplicates within the import file
- Ignored duplicates of existing properties
- Ignored duplicates of existing tax lots

#### What are the relevant tickets?
#1977 
#656 

#### Screenshots (if appropriate)
